hikey.mk

```diff
@@ -1,12 +1,28 @@
+################################################################################
+# User-defined variables
+# Edit so these match your target
+################################################################################
+# Non-secure user mode (root fs binaries): 32 or 64-bit [default 64]
 NSU ?= 64
-SU ?= 32
+# Secure kernel (OP-TEE OS): 32 or 64-bit [default 64]
 SK ?= 64
+# Secure user mode (Trusted Apps): 32 or 64-bit [default 32, requires SK=64 for 64]
+SU ?= 32
 
+################################################################################
+# Includes
+################################################################################
 -include common.mk
 
 ################################################################################
 # Mandatory definition to use common.mk
 ################################################################################
+ifeq ($(SK),32)
+ifeq ($(SU),64)
+SU = 32
```

Comment:
I'd rather $(error ...) out here instead of choosing one resolution for the conflict (why not choose `SK=64`?) -- especially since it is done silently.